components/ProfileSettings/ProfileSettings.module.css

```diff
@@ -0,0 +1,12 @@
+.button {
+  width: 100px;
+  height: 31px;
+  background-color: #6272aa;
+  border-radius: 20px;
+}
```

Comment:
using a margin right with pixels might be a good idea so u have consistent padding from the right edge of screen